Doc/library/shlex.rst

```diff
@@ -49,11 +49,13 @@ The :mod:`shlex` module defines the following functions:
    .. versionadded:: 3.8
 
 
-.. function:: quote(s)
+.. function:: quote(s, always=False)
```

Comment:
For forwards compatibility's sake? (In case new kwargs are added to `quote`?). I'm not sure if this is necessary for any backwards compatibility's sake; I've had a look at PEP-3102, but I'd be fine with this addition.

Reply:
Addressed in 9e9eff0047d731a42f972e5ee20eec6d30503a30